src/main/java/entities/Chassis.java

```diff
@@ -0,0 +1,56 @@
+package entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@Data
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = PRIVATE)
+public class Chassis {
+    @Id
+    @SequenceGenerator(name="chassis_seq",
+            sequenceName="chassis_id_seq",
+            allocationSize=1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE,
+            generator="chassis_seq")
+    int id;
```

Comment:
Why ```id``` field in **Ammunition.java** is private and here is public?

Reply:
We have '@FieldDefaults(level = PRIVATE)' annotation on our classes, so we don't need to write 'private' before type, lombok adds it automatically on compile stage. I will delete it from Ammunition